index.html

```diff
@@ -5,9 +5,51 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b89b7c8adf.js" crossorigin="anonymous"></script>
 </head>
 <body>
+
+  <main>
+
+    <nav>
+      <div class="logo"><h2>AS Pathak</h2></div> 
+      <img class="menu" src="media\Union.png" alt="hamburger" />
+    </nav>
+
+    <div class="introduction">
+      <div class="solid-welcome"><h1>I&apos;m Adarsh Glad to see you!</h1></div>
+      <div class="description">
+        <p>
+          I&apos;m a software developer! I can help you 
+          build a product , feature or website Look through some of my work and 
+          experience! If you like what you see and have a project you need coded, 
+          don&apos;t hestiate to contact me.
+        </p>
+      </div>
+
+      <div class="connect">
+        <h2><a href="mailto:adi8090808766@gmail.com" target="_blank"></a>Let&apos;S CONNECT</h2>
+      </div>
+        
+      <div class="social">
+        <a href="https://github.com/PowerLevel9000" target="_blank">
+          <i class="fa-brands fa-github"></i>
+          </a>
+        <a href="https://instagram.com/adarsh8090adi?igshid=YmMyMTA2M2Y=" target="_blank">
+          <i class="fa-brands fa-instagram"></i>
+        </a>
+        <a href="https://www.facebook.com/profile.php?id=100010279544328" target="_blank">
+          <i class="fa-brands fa-facebook"></i>
+        </a>
+        <a href="https://youtube.com/@code_gamer2.068" target="_blank">
+          <i class="fa-brands fa-youtube"></i>
+        </a>
+         
+      </div>
```

Comment:
I suggest you change this from a `div` tag to a list  that is a `ul` tag to group them, and `li` per icon link
It is important to use correct semantic tags to ensure that screen readers don't get confused when reading your page.

Reply:
thanks for your overall suggestions i just have a slip of mind i know these things about header footer main section  footer
but compleltey forget about  its implimentation  in my project